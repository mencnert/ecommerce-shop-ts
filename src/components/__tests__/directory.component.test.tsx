import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Directory from "../directory/directory.component";

it("should match snapshot", () => {
  const wrapper = shallow(<Directory />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
