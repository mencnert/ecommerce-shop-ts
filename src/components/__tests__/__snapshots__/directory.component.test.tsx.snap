// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="directory-menu"
>
  <withRouter(MenuItem)
    id={1}
    imageUrl="https://i.ibb.co/cvpntL1/hats.png"
    key="1"
    linkUrl="shop/hats"
    title="hats"
  />
  <withRouter(MenuItem)
    id={2}
    imageUrl="https://i.ibb.co/px2tCc3/jackets.png"
    key="2"
    linkUrl="shop/jackets"
    title="jackets"
  />
  <withRouter(MenuItem)
    id={3}
    imageUrl="https://i.ibb.co/0jqHpnp/sneakers.png"
    key="3"
    linkUrl="shop/sneakers"
    title="sneakers"
  />
  <withRouter(MenuItem)
    id={4}
    imageUrl="https://i.ibb.co/GCCdy8t/womens.png"
    key="4"
    linkUrl="shop/womens"
    size="large"
    title="womens"
  />
  <withRouter(MenuItem)
    id={5}
    imageUrl="https://i.ibb.co/R70vBrQ/men.png"
    key="5"
    linkUrl="shop/mens"
    size="large"
    title="mens"
  />
</div>
`;
